kinds/structured/team/team_kind.cue

```diff
@@ -0,0 +1,26 @@
+package kind
+
+name: "Team"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			// TODO docs
+			{
+				id?: int
+				orgId?: int @grafanamaturity(MaybeRemove)
```

Comment:
Are the ids supposed to be optional?

Reply:
It should be removed from the schema, probably.